services/audit-service/src/main/java/datawave/microservice/audit/config/AuditProperties.java

```diff
@@ -24,4 +28,42 @@ public long getConfirmAckTimeoutMillis() {
     public void setConfirmAckTimeoutMillis(long confirmAckTimeoutMillis) {
         this.confirmAckTimeoutMillis = confirmAckTimeoutMillis;
     }
+    
+    public Retry getRetry() {
+        return retry;
+    }
+    
+    public void setRetry(Retry retry) {
+        this.retry = retry;
+    }
+    
+    public static class Retry {
+        private int maxAttempts = Integer.MAX_VALUE;
```

Comment:
should this be a smaller number to rule out a crazy number of interrupted exceptions that causes a constant spam?

Reply:
Sure.  Done!